backend/app/Http/Controllers/AttemptController.php

```diff
@@ -54,6 +54,8 @@ public function store(Request $request)
                 'question_no' => $request->question_nos[$index],
                 'answer' => $answer,
             ]);
+
+            $index++;
```

Comment:
there is a laravel eloquent called upsert, maybe it can help.
Ref: https://laravel.com/docs/9.x/eloquent#upserts